scripts/testing/hf_serverless_smoke.py

```
@@ -0,0 +1,51 @@
+    for text in TEST_CASES:
+        payload = {"inputs": text}
+        t0 = time.time()
+        try:
+            r = requests.post(API_URL, headers=HEADERS, data=json.dumps(payload), timeout=60)
+        except Exception as e:
+            print(f"❌ Request failed for input='{text[:40]}...': {e}")
+            ok = False
+            continue
+        dt = (time.time() - t0) * 1000
+        print("—" * 40)
+        print(f"Input: {repr(text)}")
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)